File: MODUL 4/Python/PRAK402-2310817220011-ReginaSilvaMaharatini.py
bil = int(input("Input\n"))

for i in range(1, bil + 1):
    if i % 2 != 0:
        print(i, end=' ')

print()

for i in range(bil, 0, -1):
    if i % 2 == 0:
        print(i, end=' ')

print()

File: MODUL 4/C/PRAK402-2310817220011-ReginaSilvaMaharatini.c
#include <stdio.h>
int main() {
    int bil;
    printf("Input\n");
    scanf("%d", &bil);
    for (int i = 1; i <= bil; i++) {
        if (i % 2 != 0) {
            printf("%d ", i);}}
    printf("\n");
    for (int i = bil; i >= 1; i--) {
        if (i % 2 == 0) {
            printf("%d ", i);}}
    printf("\n");
 return 0;}
